### PR TITLE
parquet-converters: add 0.1

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: var/spack/repos/builtin/packages/parquet-converters/package.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ class ParquetConverters(CMakePackage):
     url      = "ssh://bbpcode.epfl.ch/building/ParquetConverters"
 
     version('develop', git=url, preferred=True)
+    version('0.1', git=url, tag='v0.1')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/parquet-converters/package.py]: wonder what should be policy here: develop preferred or last released as preferred?

**Review by pramodk — APPROVED** [review]

File: var/spack/repos/builtin/packages/parquet-converters/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/parquet-converters/package.py]: How will the default modules be built? Do we fix a version? If we just build the default, I would switch it over to the tag.

Comment (pramodk) [reply var/spack/repos/builtin/packages/parquet-converters/package.py]: Whatever is mentioned “preferred” will be deployed.

Comment (pramodk) [reply var/spack/repos/builtin/packages/parquet-converters/package.py]: I mean, we should deploy the "fixed" version.

* during development or nightly builds, we can explicitly build `@develop` to make sure latest version isn't breaking anything
* with new release, we tag new version and deploy it

Comment (matz-e) [reply var/spack/repos/builtin/packages/parquet-converters/package.py]: In that case, I think that `0.1` should be "preferred".

Comment (matz-e) [reply var/spack/repos/builtin/packages/parquet-converters/package.py]: OK, fixed

**Pull request merged**